pkg/crypto/crypto.go

```diff
@@ -35,6 +35,7 @@ var versions = map[string]uint16{
 	"VersionTLS10": tls.VersionTLS10,
 	"VersionTLS11": tls.VersionTLS11,
 	"VersionTLS12": tls.VersionTLS12,
+	"VersionTLS13": tls.VersionTLS13,
```

Comment:
@deads2k this won't make you happy.

 I updated go version to `1.12` and that made `TestConstantMaps` test to fail. It seems like a similar test exists upstream https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag_test.go#L77 which is broken as well but only if you run that test manually. Running via `make tests` don't reveal anything. I'll try to fix that.